dynamiqs/smesolve/smesolve.py

```diff
@@ -91,31 +107,240 @@ def smesolve(
         tutorial for more details.
 
     Quote: Running multiple simulations concurrently
-        The Hamiltonian `H`, the jump operators `jump_ops` and the initial density
-        matrix `rho0` can be batched to solve multiple SMEs concurrently. All other
-        arguments are common to every batch. See the
-        [Batching simulations](../../tutorials/batching-simulations.md) tutorial for
-        more details.
+        The Hamiltonian `H` and the initial density matrix `rho0` can be batched to
+        solve multiple SMEs concurrently. All other arguments are common to every batch.
+        See the [Batching simulations](../../tutorials/batching-simulations.md)
+        tutorial for more details
+
+    Warning:
+        Batching on `jump_ops` and `etas is not yet supported, if this is needed don't
+        hesitate to
+        [open an issue on GitHub](https://github.com/dynamiqs/dynamiqs/issues/new).
+
+    Warning:
+        For now the same key is used for every batched simulation over `H` and `rho0`.
+        This will be probably change in the future.
```

Comment:
Isn't this just a matter of splitting the user keys before vmapping?

Reply:
How about both?

By default, the user passes a single key, which we split and distribute among batches (with a different key per batch: seems the most natural to me and what users will expect). And we also allow for passing an array of keys of the same shape as the batch, in which case we don't do the internal splitting and directly pass each key to its corresponding batch.

This second part can come in a separate PR, later on.